sql/type.go

```diff
@@ -567,7 +567,7 @@ func IsByteType(t Type) bool {
 	}
 }
 
-// IsBinaryType checks if t is BINARY, VARBINARY, or BLOB
+// IsBinaryType checks if it is BINARY, VARBINARY, or BLOB
```

Comment:
`t` is the proper text here, we're referring to the variable. Don't think this change is needed.
Same for the change on line 560.